client/src/components/DisplayTable.js

```diff
@@ -111,7 +111,7 @@ export class DisplayTable extends React.Component {
           .value()
       )
       .thru((unsorted_array) => {
-        if (sort_by) {
+        if (col_configs_with_defaults[sort_by]) {
```

Comment:
Could also be `_.has(col_configs_with_defaults, sort_by)`. In this case, I think they're equivalent, just pointing out because using `_.has` to check existence of a key on an object can be useful in other cases (e.g. if the value of the key itself might be falsey, but all you care about is if the object has the key). Maybe worth using here just because it is a better representation of that the check actually cares about (the existence of the column, not the value of the config).